components/deck/WordCard.js

```diff
@@ -42,9 +42,9 @@ const Seperator = styled.hr`
 export default ({ card }) => {
   return (
     <WordCard>
-      <Word>{card.original ? card.original : 'Original Word'}</Word>
+      <Word>{card.original ? card.original : ''}</Word>
```

Comment:
I think this is more readable

```suggestion
      <Word>{card.original}</Word>
```